src/main.ts

```diff
@@ -1,9 +1,19 @@
 import { NestFactory } from '@nestjs/core';
-import { AppModule } from './app.controller';
+import { SwaggerModule, DocumentBuilder } from '@nestjs/swagger';
+import { AppModule } from './app.module';
 
 async function bootstrap() {
   const app = await NestFactory.create(AppModule);
+
+  const config = new DocumentBuilder()
+    .setTitle('Me Salva')
+    .setDescription('API que documenta os endpoints da aplicação Me Salva')
```

Comment:
Esqueci de comentar, tudo relacionado tudo que for relacionado a comentarios nomes de endpoint e de funçoes sera em ingles por padrão mesmo.